### ISSUE TITLE
How to install/run? Can it be installed/runned on Mac? Or does it have to be on a webserver?

### ISSUE THREAD

**Issue by Gosasan** [kwuenscher/gdax-hotkey-marketmaker#1]
Hi,

I'm completely new to python and I was wondering how to get this to work. Do I have to run in locally on my Mac or does this have to be copied to a webserver?

I'm looking forward to try it out!

Thanks!

**Comment by kwuenscher** [comment]
Hi there, 
Just updated some things, haven't run this in ages. Just fill in you API keys and then run 

python keyboard_trade.py 

If you're using macOS then you have to run it with sudo. 

Have fun!

**Comment by Gosasan** [comment]
Hi, thanks for getting back at me so quickly. I downloaded the master-zip file and unpacked. I upgraded python to 3.6.4 just in case on my mac.

I live in europe and use EUR on GDAX. Should I do a find a replace in all the files where it says BTC-USD and replace it by BTC-EUR?

Do you still used it yourself :-)
Thnx!

**Comment by Gosasan** [comment]
I think I'm a step closer.
I changed gdax.gdax_auth to gdax_auth so I suppose the local file is then used(?) and I get the following:

```
$ python3 keyboard_trade.py
Danger Zone!
initialising strategy
Traceback (most recent call last):
  File "keyboard_trade.py", line 136, in <module>
    loop.run_in_executor(None, update_balances)
NameError: name 'update_balances' is not defined
```

I don't see where update_balances is used anywhere else but I don't know python so I'm sure of this variable needs to be defined first or something.

I'm almost there :-)

**Comment by kwuenscher** [comment]
Yes, I was using the gdax python module for some reason. Also, I removed the ´´´update_balances()´´´ function from the script and forgot to delete this line. You can just remove this async loop call i.e. line 136. 

Also if you want to trade a different pair then you only have to change the product_id variable in the keyboard_trader.py script. 

No I don't really use this script anymore. I just made it so I can get a better feel for the order book.
